src/index.html

```diff
@@ -9,6 +9,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="moyo-container">
+      <header class="main-header">
+        <img src="images/logo.png" alt="moyo-logo1"
+         class="moyo-logo1">
+        <nav class="main-nav">
+          <ul class="main-nav-list">
+
+            <li class="menu-list">
+
+              <a href="#" class="is-active" >Apple</a>
+
+            </li>
```

Comment:
```suggestion
        <nav class="main-nav">
          <ul class="main-nav-list">
            <li class="menu-list">
              <a href="#" class="is-active" >Apple</a>
            </li>
```
Don't leave empty lines between parent and child elements